gobblin-config-management/gobblin-config-core/build.gradle

```diff
@@ -12,10 +12,14 @@ apply plugin: 'java'
 
 dependencies {
     compile project(':gobblin-api')
+    compile project(':gobblin-utility')
 
     compile externalDependency.slf4j
     compile externalDependency.log4j
+    compile externalDependency.lombok
     compile externalDependency.typesafeConfig
+    compile externalDependency.commonsLang3
```

Comment:
can we use `commonsLang` instead of `commonsLang3`? we should stop mixing them...